src/App.tsx

```diff
@@ -87,4 +81,22 @@ export default function App() {
       <div className="App-body">{tabs[currentTab]}</div>
     </div>
   );
+
+  // HELPERS
+  function getGSheetData() {
```

Comment:
ça me fait toujours bizarre d'avoir une fonction définie après le return!

Reply:
Adding to general refactor backlog (once chart v1 is in).